package.json

```diff
@@ -11,7 +11,7 @@
     "@craco/craco": "^6.0.0",
     "@okta/okta-react": "^3.0.2",
     "@okta/okta-signin-widget": "^4.1.2",
-    "antd": "^4.18.4",
```

Comment:
oof -- hate to see an update happen just as you push new code. Bad luck

Reply:
rip